app/views/candidate_interface/application_form/submit_show.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :title, page_title('Submit application') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_review_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <fieldset class="govuk-fieldset">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl govuk-!-margin-bottom-8">
+        <h1 class="govuk-fieldset__heading">
+          Submit application
+        </h1>
+      </legend>
+
+      <p class="govuk-body">
```

Comment:
Are we planning to put this content in locale files?

Reply:
good point, I think we agreed (tell me if I am wrong) that we should use the locale only headers, titles, button content.

For sure I need to put on locales: (link)
*Disclosure and Barring Check*
*Learn more about telling an employer about your conviction* (link)
*Submit application* (button)
...

that what you mean right?
(I don't think we should put the entire content of the page/paragraphs)

Reply:
This content should be fine to keep in the template I think! As Alessandro says, so far we've opted to use locales for stuff if we refer to it inside tests.